common/djangoapps/philu_commands/management/commands/send_ondemand_reminder_emails.py

```diff
@@ -0,0 +1,191 @@
+        Send reminder emails to those users who haven't completed the scheduled graded module for 10 days.
+        This email will not be sent for those module which don't have at-least one graded sub-section.
+    """
+
+    def handle(self, *args, **options):
+
+        # Getting all self paced courses.
+        courses = CourseOverview.objects.filter(self_paced=True, end__gte=today)
+
+        for course in courses:
+            try:
+                course_struct = CourseStructure.objects.get(course_id=course.id).structure
+            except CourseStructure.DoesNotExist:
+                log.error('Course doesn\'t have a proper structure.')
+                raise
+
+            ora_blocks = get_all_oras(course_struct)
+
+            # If course doesn't have any ORA blocks, continue.
+            if not ora_blocks:
+                continue
+
+            course_blocks = course_struct['blocks']
+
+            graded_oras_count = get_graded_ora_count(ora_blocks)
+            last_module_oras = get_last_module_ora(course_blocks)
+
+            # Getting all enrollments of user in self paced course.
+            enrollments = CourseEnrollment.objects.filter(course_id=course.id, is_active=True)
+
+            for enrollment in enrollments:
+                user = enrollment.user
+
+                try:
+                    anonymous_user = get_user_anonymous_id(user, course.id)
+                except Exception as error:
+                    log.info(error)
+                    continue
+
+                # If user hasn't enable email preferences for on demand course, no need to go further.
+                if not get_email_pref_on_demand_course(user, course.id):
+                    continue
+
+                course_chapters = modulestore().get_items(
+                    course.id,
+                    qualifiers={'category': 'course'}
+                )
+
+                course_deadline = get_suggested_course_deadline(enrollment.created.date(), course_chapters[0].children)
+
+                # Get all user submission in descending order by date
+                response_submissions = Submission.objects.filter(
+                    student_item__student_id=anonymous_user.anonymous_user_id,
+                    student_item__course_id=course.id.to_deprecated_string()).order_by('-created_at')
+
+                # Check if user has submitted last modules graded oras or not. If yes not need to send email OR
+                # If user's submission gets equal to graded oras count than don't need to continue..
+                if (last_module_oras and check_for_last_module_submission(last_module_oras, anonymous_user)) or \
+                        len(response_submissions) == graded_oras_count:
+                    continue
+
+                latest_submission = response_submissions.first()
+                # Count to keep track of how many times user shows an inactivity for "INACTIVITY_REMINDER_DAYS"
+                time_passed_count = 0
```

Comment:
since we're doing nothing with the count, we can change this into a boolean. Lets accomodate this change into the comment as well. Also change the name accordingly
Moreover lets define nearer to where its being used.